docs/modules/ROOT/pages/task.adoc

```diff
@@ -0,0 +1,4 @@
+= ec command line task documentation
```

Comment:
```suggestion
= Enterprise Contract Tekton Tasks
```

We'll have another one in [HACBS-1765](https://issues.redhat.com/browse/HACBS-1765), and possibly different versions in the future.

Reply:
ack